### DISCUSSION TITLE
coco predict

### DISCUSSION THREAD

**Discussion by w615933060** [obss/sahi#682] [Q&A] [answered]
When I make predictions on the dataset, the resulting json file has only one line. Like this,
[{"image_id":1,"bbox":[2718,1886,133,60],"score":0.9997366070747375,"category_id":2,"category_name":"jyz_dxdju_xs","segmentation":[],"iscrowd":0,"area":7980},{"image_id
Is this normal?

**Comment by fcakyon** [comment, marked as answer]
yes this format is normal. you can change the threshold to increase the number of detections.
